app-server/datamodel/src/main/scala/com/azavea/rf/datamodel/latest/schema/schema.scala

```diff
@@ -212,6 +212,91 @@ trait tables {
   /** Collection-like TableQuery object for table ScenesToBuckets */
   lazy val ScenesToBuckets = new TableQuery(tag => new ScenesToBuckets(tag))
 
+  /** Entity class storing rows of table Thumbnails
+   *  @param id Database column id SqlType(uuid), PrimaryKey
+   *  @param createdAt Database column created_at SqlType(timestamp)
+   *  @param modifiedAt Database column modified_at SqlType(timestamp)
+   *  @param organizationId Database column organization_id SqlType(uuid)
+   *  @param createdBy Database column created_by SqlType(varchar), Length(255,true)
+   *  @param widthPx Database column width_px SqlType(int4)
+   *  @param heightPx Database column height_px SqlType(int4)
+   *  @param size Database column size SqlType(size)
+   *  @param scene Database column scene SqlType(uuid), Default(None)
+   *  @param url Database column url SqlType(varchar), Length(255,true) */
+  case class ThumbnailsRow(id: java.util.UUID, createdAt: java.sql.Timestamp, modifiedAt: java.sql.Timestamp, organizationId: java.util.UUID, createdBy: String, widthPx: Int, heightPx: Int, size: String, scene: Option[java.util.UUID] = None, url: String)
```

Comment:
These generated schemas ned to be updated, looks like they're still referencing `ThumbnailsToScenes`.